pyxem/utils/expt_utils.py

```diff
@@ -569,6 +569,103 @@ def find_beam_offset_cross_correlation(z, radius_start=4, radius_finish=8):
     return (shift - 0.5)
 
 
+def calc_radius_with_distortion(x, y, xc, yc, asym, rot):
+    """ calculate the distance of each 2D points from the center (xc, yc) """
+    xp = x * np.cos(rot) - y * np.sin(rot)
+    yp = x * np.sin(rot) + y * np.cos(rot)
+    xcp = xc * np.cos(rot) - yc * np.sin(rot)
+    ycp = xc * np.sin(rot) + yc * np.cos(rot)
+
+    return np.sqrt((xp - xcp)**2 + asym * (yp - ycp)**2)
+
+
+def call_ring_pattern(xcentre, ycentre):
```

Comment:
This isn't new code?

Reply:
Yep, it's in `calibration_utils` on the master branch - should be deleted from this file - that ought to fix the coverage.